### PR TITLE
Added link to Slack channel under Developer Relevant Links

### PR DESCRIPTION
**Pull request by reddymahendra52**
Signed-off-by: reddymahendra52 reddymahendra52@gmail.com

**Screenshot:**

Screenshot (20)_LI

### CODE REVIEW

**Review by nate-double-u — COMMENTED** [review]
Hi @reddymahendra52, thanks for this!

I've made a comment about the link link used in line. Before we approve this tho, I'm wondering if this is the best place to link Slack for this project, there are a couple slack channels, so it may be better added to the community page.

@SteveLasker, thoughts?

Also, in order to merge PRs, all the commits need to be signed as per the DCO. You can see how to fix this issue here: https://github.com/notaryproject/notaryproject.dev/pull/66/checks?check_run_id=4277781799

File: config.yaml
Comment (nate-double-u) [inline config.yaml:154]: Linking to the Notary V2 channel may be better than linking to a specific thread here:
https://cloud-native.slack.com/archives/CQUH8U287

Comment (reddymahendra52) [reply config.yaml:154]: Okay
I have fixed the DCO.
I will fix the link also once it is approved from @SteveLasker .

**Comment by reddymahendra52** [comment]
Signed-off-by: reddymahendra52 reddymahendra52@gmail.com

I have also tried to resolve issue #61 by implementing the potential solutions mentioned in the respective issue.

## Here are the Results:
### Docs:
Screenshot (22)


### Blog:
Screenshot (23)


### Docs:
Screenshot (24)

**Review by nate-double-u — CHANGES_REQUESTED** [review]
Hi @reddymahendra52, thanks for fixing the DCO!

I'd recommend splitting this PR into 2.

Generally it's best practice to solve one issue per PR. Having one PR per issue helps with review (comments are only about one issue or fix), and it helps to make sure that things aren't blocked by unrelated things. For example, in this case, you mentioned that you're waiting on feedback from @SteveLasker, but if he asks for a different solution entirely, then part of this PR becomes a blocker for the rest of it and needs to be picked out.

There are times when a single PR can solve several issues, but I only recommend that when those issues are very closely related.

File: config.yaml
Comment (SteveLasker) [reply config.yaml:154]: yup, i was going to say the same thing. I just needed to get to a full computer to verify the link

Comment (SteveLasker) [reply config.yaml:154]: Actually, reading @nate-double-u 's comments, I would agree. Centralizing these in the community page is better as it becomes fewer places to update as things evolve. And, they always do.
Maybe, a "community" link, or "get involved" link that points to the community page?

**Comment by SteveLasker** [comment]
Thanks, @reddymahendra52 for the PR. As @nate-double-u suggests, splitting things up, while annoying, is a best practice to get specific feedback on the specific thing. 
A side benefit is it does give you more merged commits. :)

File: config.yaml
Comment (reddymahendra52) [reply config.yaml:154]: @SteveLasker @nate-double-u 
Can you please check if this works for you all or not??
Recording link: https://www.loom.com/share/b814f3b384474028aac6e8d7d56a0bb4

**Comment by FeynmanZhou** [comment]
@reddymahendra52 This PR has been stale for several months so we closed it. You can reopen and update it if needed. Thanks

**Pull request closed without merging**
